includes/admin/views/builder/settings-fields.php

```diff
@@ -1,9 +1,11 @@
 <?php
 /**
- * Model Field Settings Template
+ * Model Field Settings Template.
  *
- * @since   3.17.0
- * @version 3.24.0
+ * @since 3.17.0
+ * @since [version] Added support for 'hidden', 'disabled' and 'switch-multi-input' field types.
+ *
+ * @version [version]
```

Comment:
```suggestion
 * @since 3.17.0
 * @since 3.24.0 Unknown.
 * @since [version] Added support for 'hidden', 'disabled' and 'switch-multi-input' field types.
 * @version [version]
```